src/DynamoCore/Graph/Nodes/PortModel.cs

```diff
@@ -309,8 +323,11 @@ public bool IsConnected
                 return Connectors.Any() || (UsingDefaultValue && DefaultValue != null);
             }
         }
+
         #endregion
-        
+
+        internal bool IsProxyPort { get; set; } = false;
```

Comment:
If we add this property here, should we remove the one in PortViewModel?

Reply:
yes, that is already removed.

Reply:
Great, just saw it